Feature/full site

Wikitelmex opened the pull request:
# Summary
- add routing
- add home page
- add quote page
- add calculator page
- add navbar
- responsive design
- styling
- fit calculator size

wikitelmex1 committed:
Improvements

- add horizontal Nav
- add routing
- add home, calc, quotes pages

wikitelmex1 committed:
Summary

- styling improvements
- add content to pages
- fix the size of calculator to cover better the page

wikitelmex1 committed:
Summary

- styling improvements
- fix spelling

wikitelmex1 committed:
Linters

- fix linters

henatan99 commented on src/components/NavMenu/NavMenu.js:
```diff
@@ -0,0 +1,50 @@
+    <br />
+    <br />
+    <br />
+    <Switch>
+      <Route path="/Calculator">
+        <CalculatorPage />
+      </Route>
+      <Route path="/Quote">
+        <QuotePage />
+      </Route>
+      <Route path="/">
+        <HomePage />
+      </Route>
+    </Switch>
```
- [x] It is semantically wrong to render the different page components inside a NavMenu component. I recommend you move the Routes to a top-level component preferably the `App` component. :pray:

Wikitelmex replied:
I don't agree with this, since even the react tutorial that Microverse brought us is made in this exact way, however, I made the changes you required.

henatan99 reviewed (changes requested):
Hi @Wikitelmex , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
- [x] Please try to fix console errors. Your app runs well but you should always make sure that it throws no console errors. :pray: 

![Screenshot from 2021-11-03 00-11-46](https://user-images.githubusercontent.com/64894927/139952336-36661e4e-41d9-49fc-9fb3-9313ea251ec8.png)

_Check the comments under the review._

## Optional suggestions
- [x] OPTIONAL: Kindly try to make your commit messages more descriptive. It helps track the changes made easily. :pray: 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

wikitelmex1 committed:
Fix Console error

- add a void function to the input to avoid the console log error about that
- removes from the index.html, the call to manifest.json to avoid the console log error

wikitelmex1 committed:
Navbar fit to changes required

- destroy navbar component to put directly code inside app.js to be "semantically correct"

eliasecasta reviewed (approved):
## Status:  Approved 🟢

Hi, @Wikitelmex, 

Normally, non-descriptive commit messages are a reason for invalidation but the past CR marked them as optional so I will approve this project. Good luck!

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

![](https://media3.giphy.com/media/DnMMGxEvniha7CvASq/giphy.gif)

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
